frontend/src/components/Layouts/Header/Components/BurgerMenu/OpenMenu.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import styles from './BurgerMenu.module.scss';
+import Button from '../../../../ui-kit/components/Button';
+import { ICONS } from '../../../../ui-kit/icons';
+import Divider from '../../../../ui-kit/components/Divider';
+import ROUTES from '../../../../../constants/routers';
+import IconButton from '../../../../ui-kit/components/IconButton';
+import NavigationMenu from '../NavigationMenu';
+import { navigationItems } from '../../navigationItems';
+
+const OpenMenu = ({ toggleMenu }) => {
+  return (
+    <div className={styles.openMenu}>
+      <NavLink to={ROUTES.home} className={styles.containerTop}>
+        <img className={styles.logo} src="images/logo.svg" />
+        <IconButton icon={<ICONS.closeMobile />} onClick={toggleMenu} />
```

Comment:
Put it behind NavLink, because when I close the menu I am taken to the main page